DentOS_Framework/DentOsTestbedLib/src/dent_os_testbed/lib/os/recoverable_sysctl.py

```diff
@@ -0,0 +1,43 @@
+from dent_os_testbed.lib.os.sysctl import Sysctl
+
+
+class RecoverableSysctl(Sysctl):
+
+    saved_values = dict()
+
+    async def set(*argv, **kwarg):
+        devices = kwarg['input_data']
+        kwarg['parse_output'] = True
+        saved_options = {}
+        for device in devices:
+            for device_name in device:
+                saved_options[device_name] = []
+                for command in device[device_name]:
+                    saved_options[device_name].append(command.get("options", ""))
+                    command["options"] = ""
+        out = await Sysctl.get(*argv, **kwarg)
+        for device in devices:
+            for device_name in device:
+                if device_name in RecoverableSysctl.saved_values:
+                    for variable, value in out[0][device_name]['parsed_output'].items():
```

Comment:
Now there is a different problem: if the user passes `-n` then the return value will be of type `list`, and `list` doesn't have an `items()` method

Reply:
That's why I added removal of "options" param on lines 12-17